README.md

```diff
@@ -7,7 +7,14 @@ Microsoft Authentication Library (MSAL) Preview for Android
 The MSAL library for Android gives your app the ability to begin using the [Microsoft Cloud](https://cloud.microsoft.com) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. The library also supports [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/).
 
 [![Version Badge](https://img.shields.io/maven-central/v/com.microsoft.identity.client/msal.svg)](http://repo1.maven.org/maven2/com/microsoft/identity/client/msal/)
-[![Build Status](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android.svg?branch=master)](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android)
+
+## Build Status
+| Branch  | Status |
+| ------------- | ------------- |
+| dev (Travis)  | [![Build Status](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android.svg?branch=master)](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android) |
+| dev (VSTS)  | [![Build status](https://identitydivision.visualstudio.com/_apis/public/build/definitions/a7934fdd-dcde-4492-a406-7fad6ac00e17/126/badge)](https://identitydivision.visualstudio.com/IDDP/_build/index?context=allDefinitions&path=%5CCI&definitionId=126&_a=completed) |
+
+Note: A corpnet account is required to view the VSTS build.
```

Comment:
nit: We could specify the 'Microsoft corpnet' -- I think this is pretty minor, though. The rest LGTM 👍